test/unit/browser/System.js

```diff
@@ -83,7 +83,7 @@ suite('System.js', function() {
 
   test('System.set', function() {
     var store = $traceurRuntime.ModuleStore;
-    var polyfills = store.getForTesting('src/runtime/polyfills/polyfills.js');
+    var polyfills = store.get(System.version + '/src/runtime/polyfills/polyfills.js');
```

Comment:
I could not figure out how to test this one. It gets reference from test/runtime.html but that is completely broken.